### PR TITLE
PCHR-3436: Use Datepicker and from shoreditch

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this PR local copy of datepicker styles has been deleted from Job Contract and instead the styles from Shoreditch has been used to maintain consistency and reduce redundancy.

## Technical Details
1. `hrjobcontract/scss/civihr/partials/_datepicker.scss` has been removed.
2. Text angular styles has been removed from `hrjobcontract/scss/civihr/partials/_forms.scss`.
3. Uib-Datepicker has been imported from shoreditch
```scss
@import '../../../org.civicrm.shoreditch/scss/bootstrap/overrides/variables';
@import '../../../org.civicrm.shoreditch/scss/bootstrap/mixins/prefix';
@import '../../../org.civicrm.shoreditch/scss/angular/overrides/uib-datepicker';
```
Job Contract has a compass based build system and SCSSROOT is not integrated yet, relative paths has been used above.

## Comments
Backstop and Manual testing has been performed to ensure nothing is broken.

### CODE REVIEW

File: hrjobcontract/scss/civihr/partials/_forms.scss [hunk 1]
```diff
@@ -32,6 +35,7 @@ input[type="checkbox"],
   }
 
   .has-feedback {
+    /* stylelint-disable selector-max-compound-selectors */
```
Comment (AkA84) [inline hrjobcontract/scss/civihr/partials/_forms.scss]: shouldn't this be `disable-line`

File: hrjobcontract/scss/hrjc.scss [hunk 2]
```diff
@@ -1,125 +1,128 @@
...
+  @import 'hrjc/partials/list';
+  @import 'hrjc/partials/loader';
+  @import 'hrjc/partials/modals';
+  @import 'hrjc/partials/navs';
+  @import 'hrjc/partials/panels';
+  @import 'hrjc/partials/summary';
+  @import 'hrjc/partials/tables';
+  @import 'hrjc/partials/wizard';
+
+  @import '../../../org.civicrm.shoreditch/scss/bootstrap/overrides/variables';
```
Comment (AkA84) [inline hrjobcontract/scss/hrjc.scss]: use SCSSROOT

File: hrjobcontract/css/hrjc.css [hunk 3]
```diff
@@ -61,6 +61,7 @@
   /* add generic styling for the editor */
   /* add the styling for the awesomness of the resizer */
   /* copy the popover code from bootstrap so this will work even without it */
+  /* stylelint-disable selector-no-qualifying-type */
```
Comment (AkA84) [inline hrjobcontract/css/hrjc.css]: why is this necessary? The .css file should not be even considered by the linter.

File: hrjobcontract/scss/civihr/partials/_forms.scss (hunk 1 above)
Comment (deb1990) [reply hrjobcontract/scss/civihr/partials/_forms.scss]: No inside this block, there are multiple instances of more than 3 compound selectors, so disabling all at once.

File: hrjobcontract/scss/hrjc.scss (hunk 2 above)
Comment (deb1990) [reply hrjobcontract/scss/hrjc.scss]: Job Contract has a `compass` based build system and SCSSROOT is not integrated yet, so did not add that.

File: hrjobcontract/css/hrjc.css (hunk 3 above)
Comment (deb1990) [reply hrjobcontract/css/hrjc.css]: This is not added manually, after build process it is simply copied.
